Microsoft.Toolkit.Uwp.UI.Controls.WinRT/UniversalWindowsAppPackageFontLoader/FontFileEnumerator.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+#include "UniversalPackageFontData.h"
+
+BEGIN_NAMESPACE_CONTROLS_WINRT
+
+namespace UniversalWindowsAppPackageFontLoader
+{
+    /// <summary>
+    /// This is an enumerator that basically just enumerates 1 custom font into DWrite from the Universal Windows App package.
+    /// It's extremely simplistic and basically just connects the Universal App Package files to DWrite.
+    /// </summary>
+    struct FontFileEnumerator : winrt::implements<FontFileEnumerator, IDWriteFontFileEnumerator>
```

Comment:
Same here, does this struct need to be exposed to the developer?

Reply:
This isn't exposed.